Source/CarthageKitTests/ProjectSpec.swift

```diff
@@ -12,9 +12,113 @@ import Nimble
 import Quick
 import ReactiveSwift
 import Tentacle
+import Result
+import ReactiveTask
+import XCDBLD
 
 class ProjectSpec: QuickSpec {
 	override func spec() {
+		describe("createAndCheckVersionFiles") {
+			let directoryURL = Bundle(for: type(of: self)).url(forResource: "DependencyTest", withExtension: nil)!
+			let buildDirectoryURL = directoryURL.appendingPathComponent(CarthageBinariesFolderPath)
+			
+			func buildDependencyTest(platforms: Set<Platform> = [], cacheBuilds: Bool = true) -> Set<String> {
+				var builtSchemes: [String] = []
+				
+				let project = Project(directoryURL: directoryURL)
+				let result = project.buildCheckedOutDependenciesWithOptions(BuildOptions(configuration: "Debug", platforms: platforms, toolchain: "com.apple.dt.toolchain.Swift_2_3", cacheBuilds: cacheBuilds))
```

Comment:
At the moment, yes it does. These tests were written before the swift 3 stuff. They were chosen because they were simple projects (i.e., don't take forever to compile), but also had nested dependencies. Unfortunately they don't have branches that compile with swift3. 

What we can do is construct a better, toolchain safe set of dependencies in the fixture and replace this test. I'll start work on getting that in place, but we're hoping it can be a follow-up PR to avoid any more conflicts / delays?